livechat/agent/web/client.py

```diff
@@ -957,4 +958,10 @@ def list_agents_for_transfer(self,
 
 
 class AgentWeb33(AgentWebInterface):
-    ''' Customer API version 3.3 class. '''
+    ''' Agent API version 3.3 class. '''
+    pass
+
+
+class AgentWeb34(AgentWebInterface):
+    ''' Agent API version 3.4 class. '''
+    pass
```

Comment:
pass is not necessary (https://www.python.org/dev/peps/pep-0257/)

Reply:
👍